config/webpack.proxy.config.js

```diff
@@ -10,7 +10,7 @@ const { _BUILD_RELATIVE_DIRNAME, DEV_BRANCH, DEV_PORT } = setupDotenvFilesForEnv
 let BETA_PREFIX = '';
 
 if (/(prod|stage|qa|ci)-beta/.test(DEV_BRANCH)) {
```

Comment:
missed this `beta` reference

Reply:
this is fine, we're leaving references to the envs as `beta`... this won't be confusing at all